src/components/PageNotFoundComponent.jsx

```diff
@@ -19,15 +19,23 @@ const PageNotFoundComponent = props => (
       id="sad-face-404"
     />
     <p className="title-404"> Sorry we could not find that page! </p>
-    <p className="para-404"> There's no shame in being lost in a page that was probably never there in the first place! </p>
-    <Button className="back-button-404" onClick={() => props.history.goBack()}>let's go back</Button>
+    <p className="para-404"> There's no shame in being lost in a page that was probably never there
+      in the first place!
+    </p>
+    <Button className="back-button-404" onClick={() => props.history.goBack()}>let's go
```

Comment:
I'm curious....why can't this be just a function passed by reference instead of a curry function? Curry function(apart from the `render` prop) are usually discouraged as a new function will be created each time the render function is executed, which may cause performance degradation in some cases
```jsx
<Button className="back-button-404" onClick={props.history.goBack}>...</Button>